packages/framework/src/handler.ts

```diff
@@ -212,6 +221,29 @@ export class NovuRequestHandler<Input extends any[] = any[], Output = any> {
     };
   }
 
+  private triggerAction(triggerEvent: ITriggerEvent) {
+    return async () => {
+      if (!triggerEvent.bridgeUrl && process.env.NEXT_PUBLIC_VERCEL_URL) {
```

Comment:
This should be provided in framework-specific handlers or `trigger` function re-exports. This is reusable logic across all frameworks so I'd remove it and respect the arguments of the `triggerEvent`.